tests/component/TracksTable.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import { test, expect } from '@playwright/experimental-ct-react';
+import { TracksTable } from '@/Components/components';
+import { TrackContextProvider } from '@/contexts/contexts';
+import { API_ENDPOINTS } from '@/lib/constants/constants';
+
+import type { Track, PaginationMeta } from '@/lib/types/types';
+
+const mockTracksSortedByTitle: Track[] = [
+  {
+    id: '1',
+    title: 'Bohemian Rhapsody',
+    artist: 'Queen',
+    album: 'A Night at the Opera',
+    slug: 'bohemian-rhapsody',
+    genres: ['Rock'],
+    createdAt: '1975-10-31T00:00:00.000Z',
+    updatedAt: '1975-10-31T00:00:00.000Z',
+  },
+  {
+    id: '2',
+    title: 'Smells Like Teen Spirit',
+    artist: 'Nirvana',
+    album: 'Nevermind',
+    slug: 'smells-like-teen-spirit',
+    genres: ['Grunge', 'Rock'],
+    createdAt: '1991-09-10T00:00:00.000Z',
+    updatedAt: '1991-09-10T00:00:00.000Z',
+  },
+];
+
+const mockTracksSortedByArtist: Track[] = [
+  {
+    id: '2',
+    title: 'Smells Like Teen Spirit',
+    artist: 'Nirvana',
+    album: 'Nevermind',
+    slug: 'smells-like-teen-spirit',
+    genres: ['Grunge', 'Rock'],
+    createdAt: '1991-09-10T00:00:00.000Z',
+    updatedAt: '1991-09-10T00:00:00.000Z',
+  },
+  {
+    id: '1',
+    title: 'Bohemian Rhapsody',
+    artist: 'Queen',
+    album: 'A Night at the Opera',
+    slug: 'bohemian-rhapsody',
+    genres: ['Rock'],
+    createdAt: '1975-10-31T00:00:00.000Z',
+    updatedAt: '1975-10-31T00:00:00.000Z',
+  },
+];
```

Comment:
Consider defining a base Track array first. This allows you to derive your specific sorted mock arrays from a single source of truth, preventing consistency issues if track details (like titles) change.